libs/functions/generate_shapefiles.py

```diff
@@ -36,34 +57,54 @@ def join_and_output_shapefile(df, shp_reader, pivot_shp_field, pivot_df_column,
             # Gets the row of the dataframe that matches the FIPS codes for a state/county
             row = df[df[pivot_df_column] == shapeRecord.record[pivot_shp_field]].iloc[0]
         except Exception as e:
-            state_fips = shapeRecord.record[pivot_shp_field][:2] # state fips is the first two chars of the state/county fips
-            failed_dictionary.setdefault(state_fips, []).append(shapeRecord.record[pivot_shp_field])
+            state_fips = shapeRecord.record[pivot_shp_field][
```

Comment:
this one hurts me

Reply:
yeah, for these I generally try to rewrite them so that it doesn't happen (in this case it would just be moving the comment on top)